production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json

```diff
@@ -1662,22 +2655,37 @@
                      }
                   },
                   "panels": [ ],
+                  "pluginVersion": "11.4.0-77663",
                   "targets": [
                      {
                         "datasource": {
                            "type": "prometheus",
                            "uid": "${datasource}"
                         },
                         "editorMode": "code",
-                        "expr": "sum by(instance, pod) (rate(node_disk_written_bytes_total[$__rate_interval]))\n+ ignoring(pod) group_right() \n(count by(instance, pod) (container_fs_writes_bytes_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"bloom-gateway\", device!~\".*sda.*\"}) * 0)",
+                        "expr": "sum(sum by (instance) (rate(node_disk_read_bytes_total[$__rate_interval]))\n+ on(instance) group_right() \n(count by (instance) (container_fs_reads_bytes_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"bloom-gateway\", device!~\".*sda.*\"}) * 0))",
                         "hide": false,
                         "instant": false,
-                        "legendFormat": "{{pod}}",
+                        "interval": "",
+                        "legendFormat": "Reads",
                         "range": true,
                         "refId": "D"
+                     },
+                     {
+                        "datasource": {
+                           "type": "prometheus",
+                           "uid": "${datasource}"
+                        },
+                        "editorMode": "code",
+                        "expr": "sum(sum by(instance) (rate(node_disk_written_bytes_total[$__rate_interval]))\n+ on(instance) group_right() \n(count by(instance) (container_fs_writes_bytes_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"bloom-gateway\", device!~\".*sda.*\"}) * 0)) * -1",
+                        "hide": false,
+                        "instant": false,
+                        "legendFormat": "Writes",
+                        "range": true,
+                        "refId": "A"
                      }
                   ],
-                  "title": "Disk writes",
+                  "title": "Disk reads/writes",
```

Comment:
With a 12h time range, this fails with 

```
execution: query processing would load too many samples into memory in query execution
``` 

It works with a 1h time range though. Should we change the query to load less samples?